AnnoyancesFilter/sections/popups.txt

```diff
@@ -4328,6 +4328,8 @@ e1.ru,ngs.ru,nn.ru,116.ru,14.ru,154.ru,161.ru,164.ru,178.ru,26.ru,29.ru,35.ru,42
 skipol.pl,nosteam.ro,gifyagusi.pl,legendas.tv,polszczyzna.pl,boardgamesonline.net,yes-icancook.com,bestel.az,nosteamgames.ro,majorgeeks.com,bestfilmeshd.com,mangakyo.net,programminggeek.in,elise.com.ua,for-ua.com,asn.in.ua,mocasoft.ro,cydiaplus.com,dtorrent.com.br,tarihkomplo.com,uptobox.com,uptostream.com###fanback
 evvelcevap.com##.snp-overlay
 evvelcevap.com##body .snp-bld-showme
+doramy.moy.su##.fancybox-desktop
```

Comment:
No need to sort rules order inside one section. Review is harder after changes in master.

Reply:
ok. I fix this PR. And resend PR as new one.